doc/api/errors.md

```diff
@@ -3302,6 +3288,46 @@ changes:
 
 An invalid transfer object was passed to `postMessage()`.
 
+<a id="ERR_IMPORT_ASSERTION_TYPE_FAILED"></a>
+
+### `ERR_IMPORT_ASSERTION_TYPE_FAILED`
+
+<!-- YAML
+added:
+  - v17.1.0
+  - v16.14.0
+removed: REPLACEME
+-->
+
+An import assertion has failed, preventing the specified module to be imported.
```

Comment:
```suggestion
An import assertion has failed, preventing the specified module being imported.
```

Reply:
I'd prefer to keep it as is, it's the wording used today in https://nodejs.org/api/errors.html#err_import_assertion_type_failed.